src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/GraphQLUserContext.cs

```diff
@@ -1,11 +1,12 @@
 using System;
+using System.Collections.Generic;
 using System.Security.Claims;
 using System.Threading;
 using Microsoft.AspNetCore.Http;
 
 namespace OrchardCore.Apis.GraphQL
 {
-    public class GraphQLContext
+    public class GraphQLUserContext : Dictionary<string, object>
```

Comment:
maybe we can remove `ServiceProvider` from this file; the service provider should always be resolved through .RequestServices`; unless some other things then fieldresolvers use this - not entirely sure why this stores the HttpContext either; that should def be removed.